engines/web/test/web_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class WebTest < ActiveSupport::TestCase
+  test "truth" do
+    assert_kind_of Module, Web
+  end
+end
```

Comment:
🔥 I guess this is just an example of a test autogenerated by Rails.

Reply:
yep, deleted.